### DISCUSSION TITLE
how to ignore or close log about webpack5 cache in terminal?

### DISCUSSION THREAD

**Discussion by lanmch** [webpack/webpack#14572] [Q&A]
when I compile my project with webpack5 of cache property. I get a lot of logs like these. How can I close it like config.ignoreWarning?
![image](https://user-images.githubusercontent.com/37408756/138832637-03519142-8598-4af7-b2e7-93d1801b4f51.png)

**Comment by alexander-akait** [comment]
You need fix these warnings,please try to update `html-webpack-plugin`

**Reply by lanmch** [reply]
Does it mean there is no way to block it here?

**Reply by lanmch** [reply]
I hav tried to update html-webpack-plugin@5.5.0 (lastest) but it doesnt work

**Reply by alexander-akait** [reply]
You can do it using https://webpack.js.org/configuration/other-options/#level, but I strongly recommend to fix it

**Reply by alexander-akait** [reply]
You can provide reproducible example and I will help

**Reply by lanmch** [reply]
I am sorry, like the old question I mentioned. I join a company and do the work of webpack. this project hav been init about 4 years, and it is too large so that i cant modify it hastily. A little change may lead to many bugs
